index.html

```diff
@@ -24,6 +24,23 @@
     />
   </head>
   <body>
+    <!-- Header Component-->
+    <header>
+      <nav class="pre-sticky">
+        <a href="/" class="header__sign-in">Iniciar Sesión</a>
+      </nav>
```

Comment:
what's the purpose of the pre-sticky element?

Reply:
ok got it, it will be used for the fade-in animation

Reply:
However, I think the styles are not correct
![Screen Shot 2022-04-20 at 8 04 20 PM](https://user-images.githubusercontent.com/36536646/164351175-c33a38b3-8d21-47f0-a545-5205086ed0a3.png)

Reply:
In the commit 7f402b I wrote a comment on how to set the display property of navbar__hero

Reply:
Thank you